test/mode/modeVisual.test.ts

```diff
@@ -5,9 +5,14 @@ import { ModeHandler } from '../../src/mode/modeHandler';
 import { setupWorkspace, cleanUpWorkspace, assertEqualLines, assertEqual } from './../testUtils';
 import { ModeName } from '../../src/mode/mode';
 import { TextEditor } from '../../src/textEditor';
+import { getTestingFunctions } from '../testSimplifier';
```

Comment:
Oh we can do this step by step. Each time we add new test cases to old ones, we can do the tweaking at the same time.